app/controllers/services_controller.rb

```diff
@@ -4,6 +4,7 @@ class ServicesController < ApplicationController
 
   def register
     changes_made = false
+    changed_service = nil
```

Comment:
The JS developer in me says "but if we don't define it out here, how will we access it later?"